src/Uno.UI/UI/Xaml/Controls/Slider/Slider.mux.cs

```diff
@@ -1534,7 +1535,7 @@ public object Convert(object value, Type targetType, object parameter, string la
 
 			roundedValue = DoubleUtil.Round(originalValue, numPlacesPastDecimalPoint);
 
-			return string.Format(szFormat, roundedValue);
+			return string.Format(CultureInfo.InvariantCulture, szFormat, roundedValue);
```

Comment:
`CurrentUICulture` this is presented to the user.

Reply:
```suggestion
			return string.Format(CultureInfo.CurrentUICulture, szFormat, roundedValue);
```